src/main/java/com/hazelcast/kubernetes/HazelcastKubernetesDiscoveryStrategy.java

```diff
@@ -103,6 +104,35 @@ private String discoverZone() {
         return "unknown";
     }
 
+    /**
+     * Discovers the name of the node which the current Hazelcast member pod is running on.
+     * <p>
+     * Note: NODE_AWARE is available only for the Kubernetes API Mode.
+     */
+    private String discoverNodeName() {
+        if (DiscoveryMode.KUBERNETES_API.equals(config.getMode())) {
+            try {
+                String podName = System.getenv("POD_NAME");
+                if (podName == null) {
+                    podName = System.getenv("HOSTNAME");
+                }
+                if (podName == null) {
+                    podName = InetAddress.getLocalHost().getHostName();
+                }
+                String nodeName = client.nodeName(podName);
+                if (nodeName != null) {
+                    getLogger().info(String.format("Kubernetes plugin extracted node name: %s", nodeName));
```

Comment:
```suggestion
                    getLogger().info(String.format("Kubernetes plugin discovered node name: %s", nodeName));
```

For the sake of consistency.